### DISCUSSION TITLE
operator and variant

### DISCUSSION THREAD

**Discussion by wandth** [Autodesk/arnold-usd#879] [Q&A]
Hello，Did someone knows is the arnold usd plugin supported to export operators as material variant？

thanks

**Comment by sebastienblor** [comment]
Hi @wandth , apologies for the lack of reply, we had simply missed this "discussions" section.
This is not planned at the moment. USD Variants can do lots of things that could be hard to represent as arnold operators. 
So at the moment we loose the different variants when the scene is converted to arnold, we just consider the "current" variant.
